test/common/grpc/async_client_manager_impl_test.cc

```diff
@@ -207,21 +245,57 @@ TEST_F(AsyncClientManagerImplTest, GrpcServiceConfigWithHashKeyTest) {
             config_with_hash_key_c.getPreComputedHash());
 }
 
+TEST_F(AsyncClientManagerImplTest, RawAsyncClientCacheWithConfig) {
```

Comment:
Please add a one-line comment describing the test.

Reply:
done